examples/team0_up.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
I see no need to show a team without slaves as an example. It is a useless config.

Reply:
Done, thanks!